doc/next/_sources/campaign-budgets.rst.txt

```diff
@@ -0,0 +1,71 @@
+.. _campaign_budgets_overview:
+
+Campaign Budgets
+================
+
+Budgets are used to limit the spending for an individual campaign. Your actual spent for a campaign may be lower than the limits you set, depending on a lot of factors; however, it can never be higher.
+Campaign budgets **must be defined**, even if they are intended to be set to *unlimited*. Absence of campaign budgets means that the ads will not be live.
+When ads in this campaign receive clicks, all budgets of this campaign will be checked for remaining balance. In case balance
+runs out on any one of the configured budgets, all ads in this campaign will be taken offline until the budget(s) balance is reset or budgets are increased. The campaign will then be in status *BUDGET_REACHED*.
```

Comment:
For each click a ad receives the available budget in the campaign is checked. In case the campaign has no remaining budget available in any one of its configured budgets, all ads for this campaign will be taken offline untill the budget(s) balance(s) are reset or increased. The campaign will be marked with the status *BUDGET_REACHED*.

Reply:
`In case the campaign has no remaining budget available in any one of its configured budgets` to me formulating it this way sounds prone to mis-interpretation. 
`balance runs out on any one of the configured budgets` makes it explicit that a monthly budget being reached (while daily is techinicaly not), means also that the campaign is declared budget reached.

Reply:
Ok, just consider that we don't really use the concept of balance persee. So we might want to introduce it.

Reply:
ok, will reword that, something like `amount`

Reply:
we're not really providing balance (aka remaining budget) numbers directly, so better not introduce the term